README.md

```diff
@@ -45,7 +45,7 @@ Besides VirtualBox, provider support is also included for Parallels, Hyper-V, VM
 
 Installation instructions can be found [here](https://varyingvagrantvagrants.org/docs/en-US/installation/).
 
-Full documentation can be found in the `docs/` directory of this repository and on the [varyingvagrantvagrants.org](https://varyingvagrantvagrants.org) website.
+Full documentation can be found on the [varyingvagrantvagrants.org](https://varyingvagrantvagrants.org) website.
```

Comment:
> The management of VVV documentation has been moved to the https://github.com/Varying-Vagrant-Vagrants/varyingvagrantvagrants.org repository.

How would people who don't already know this find out about it?

How about an extra sentence in the readme about it?  "Submit documentation PRs [here](https://github.com/Varying-Vagrant-Vagrants/varyingvagrantvagrants.org).

A link in the site footer to "edit this page on GitHub" would also be lovely, but that is a separate change.

Reply:
I think having something in the readme makes sense. I also opened https://github.com/Varying-Vagrant-Vagrants/VVV/issues/1277 the other day to address `CONTRIBUTING.md`.